src/Screen.hh

```diff
@@ -94,23 +103,41 @@ public:
 
     bool isRootColormapInstalled() const { return root_colormap_installed; }
     bool isScreenManaged() const { return m_state.managed; }
-    bool isWorkspaceWarping() const { return (m_workspaces_list.size() > 1) && *resource.workspace_warping; }
```

Comment:
Not sure what the point of un-inlining these functions is?

Reply:
Well, the new ones were clearly too big to inline, and I thought they should all be consistent. But I gather you prefer making the change smaller to consistency among these methods, so in the redo I won't touch any methods not strictly necessary.